DateTimeMate.go

```diff
@@ -73,11 +75,35 @@ func removeTrailingS(s string) string {
 	return strings.TrimSuffix(s, "s")
 }
 
-// Reformat the source string to match the strftime outputFormat
-// Ex: "2024-07-22 08:21:44", "%v %r" => "22-Jul-2024 08:21:44 AM"
+// Reformat converts a date/time string into a specified format. The source can be:
+//   - A Unix timestamp (e.g., "1700265600")
+//   - A relative date (e.g., "yesterday", "now")
+//   - Any other date format parseable by parsetime
+//
+// The outputFormat parameter uses strftime format specifiers, with additional
+// support for Unix seconds via '%s'.
+//
+// Example usage:
+//
+//	s, err := Reformat("1700265600", "%Y-%m-%d")         // Unix timestamp to date
+//	s, err := Reformat("yesterday", "%Y-%m-%d %H:%M:%S") // Relative date to datetime
+//	s, err := Reformat("2024-01-01", "%s")               // Date to Unix timestamp
+//
+// Returns an error if:
+//   - The outputFormat is invalid
+//   - The source date cannot be parsed
+//   - The time parser initialization fails
 func Reformat(source string, outputFormat string) (string, error) {
-	source = ConvertRelativeDateToActual(source)
-	f, err := strftime.New(outputFormat)
+	if isPureIntegerAtoi(source) {
+		t, err := UnixStringToTime(source)
+		if err != nil {
+			return "", err
+		}
+		source = t.String()
+	} else {
+		source = ConvertRelativeDateToActual(source)
+	}
+	f, err := strftime.New(outputFormat, strftime.WithUnixSeconds('s'))
```

Comment:
Please add a comment to explain what it does, especially the WithUnixSeconds("a")

One way could to create a function that will be documented in the godoc, and here you simply call it.

Reply:
I added a code-comment for this:

```go
// creates a new Strftime instance
// outputFormat is a pattern string that follows strftime formatting
// the additional formatting behavior allows this to also use the unix time %s modifier
f, err := strftime.New(outputFormat, strftime.WithUnixSeconds('s'))
```